go/vt/vttablet/tabletmanager/vdiff/schema.go

```diff
@@ -81,12 +81,16 @@ const (
 	sqlGetVDiffByKeyspaceWorkflowUUID = "select * from _vt.vdiff where keyspace = %s and workflow = %s and vdiff_uuid = %s"
 	sqlGetMostRecentVDiff             = "select * from _vt.vdiff where keyspace = %s and workflow = %s order by id desc limit 1"
 	sqlGetVDiffByID                   = "select * from _vt.vdiff where id = %d"
-	sqlVDiffSummary                   = `select vd.state as vdiff_state, vdt.table_name as table_name,
-										vd.vdiff_uuid as 'uuid', vdt.state as table_state, vdt.table_rows as table_rows,
-										vd.started_at as started_at, vdt.rows_compared as rows_compared, vd.completed_at as completed_at,
-										IF(vdt.mismatch = 1, 1, 0) as has_mismatch, vdt.report as report
-										from _vt.vdiff as vd inner join _vt.vdiff_table as vdt on (vd.id = vdt.vdiff_id)
-										where vdt.vdiff_id = %d`
+	sqlDeleteVDiffs                   = `delete from vd, vdt using _vt.vdiff as vd inner join _vt.vdiff_table as vdt on (vd.id = vdt.vdiff_id) where
```

Comment:
we also need to purge from the `vdiff_log` table

Reply:
OK, I thought that was supposed to be some kind of permanent record (like a log). I can delete from it though.

Reply:
Done: https://github.com/vitessio/vitess/pull/10608/commits/516a12c10f59a207e48eff3a4a029fdcab30f6fb